app/src/main/res/values/styles.xml

```diff
@@ -459,4 +459,9 @@
     <item name="android:gravity">center_horizontal</item>
   </style>
 
+  <!-- ShapeableImageView -->
+  <style name="ShapeAppearanceOverlay.App.Circular" parent="">
+    <item name="cornerSize">50%</item>
+  </style>
```

Comment:
Another comment to continue the discussion from https://github.com/oppia/oppia-android/pull/4139#discussion_r796232690 since it hasn't been resolved yet.

Reply:
From your previous comment regarding omitting this parent part, I guess that's not possible since its the part of this property and omitting will give error.

Reply:
If a parent needs to be defined, then we ought to derive this from the main Oppia theme since that's also a material theme and may pre-define the cornerSize property.

Reply:
https://howtodoandroid.com/shapeableimageview-material-components-android/  
From here I read that the key to the overlay behavior is the empty parent attribute. This is what causes the overlay behavior.

Reply:
Ah, I think that should be explicitly called out then in the comment above, and why it works (since otherwise it looks like a potential mistake that may be incorrectly changed in the future).